tslint.json

```diff
@@ -4,6 +4,7 @@
   ],
   "rules": {
     "arrow-parens": false,
+    "object-literal-sort-keys": false,
```

Comment:
@WoH did you skip the husky hook or something? When I went to deploy your changes I got a bunch of linter errors. Anyway, I'm going to disable this rule because it's a real pain. And it was much faster than changing these 11+ places where the linter was throwing.

Reply:
I won't pretend I haven't skipped it in my life ever. But I did run it before in this case because I know I miss way too many semicolons and brackets due to my own stylistic preferences.

Right now, `npm run lint` still passes on my feature branch (which has fb1f5231978893c73b3fcdd22118254f4bbf209c as the latest commit) :confused:

Reply:
Are you sure you are running the packaged version of tslint and not your global installation or something?